lib/chat_channel_archive_service.rb

```diff
@@ -225,7 +226,9 @@ def notify_archiver(result, error: nil)
     )
   end
 
-  def kick_all_users
-    UserChatChannelMembership.where(chat_channel: chat_channel).update_all(following: false)
+  def kick_all_users(last_message_id)
```

Comment:
Just wonder why we need to pass this as an argument when this method can derive the `last_message_id` from the `chat_channel` object.

Reply:
good point will change